ge.h

```diff
@@ -152,12 +239,28 @@ struct ge {
 
     uint8_t AINI:1;
     /* Faults: TODO (pp. 139-141) */
+    uint8_t rFA;
 
     struct ge_console console;
+    int ge_console_socket;
```

Comment:
I don't think this should be here.
This should be internal to the console_socket implementation